tests/Controller/ClientDeleteControllerTest.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace SimpleSAML\Test\Module\oidc\Controller;
+
+use Laminas\Diactoros\Response\RedirectResponse;
```

Comment:
Just out of curiousity.. Why not use the RedirectResponse that Symfony provides? It is already available when using SSP 2.0

Reply:
Since that Diactoros package is used widely in the whole module, I left it this way. I agree it should be replaced with Symfony ones, however, I thought to do it at a time when we also move to new Symfony routing and container implementation (currently we have custom container implementation). Do you think that we should do this now? I'm afraid this could considerably postpone the release of new version of this module... Also, @pradtke what do you think? Should we move to Symfony routing / container now or leave it for later? Module currently works with SSPv2 as is...

Reply:
I have no preference, I was just curious

Reply:
@cicnavi I think we leave it as is. I agree that doing more changes would just delay finalizing the SSPv2 version. We can do further cleanup in another release.